src/main/java/com/example/planit/model/mongo/holiday/Holiday.java

```diff
@@ -0,0 +1,37 @@
+package com.example.planit.model.mongo.holiday;
+
+import org.springframework.data.mongodb.core.mapping.Document;
+import org.springframework.data.mongodb.core.mapping.Field;
+
+@Document("holidays")
+
+public class Holiday {
+    @Field(name = "name")
+    private String holidayName;
+    @Field(name = "date")
+    private String holidayStartDate;
```

Comment:
The date is an ISO string? The other full day events are represented with the Google `Event` object, where the date is a `DateTime`. I think that is should be consistant because the front gets just a list of full day events represented by Google `Event`s.
So @Danielsio what do you think is better:
1. Save the Holidays as Google `Event`s in the DB.
2. Convert the Holidays from `Holiday` object to Google `Event`, right before we merge the fullDayEvents list and the holidays list.

Reply:
i like simple solutions, so options number 2 is my choice.

iso string are ez to convert -> Instant, DateTime etc..

option number 1 requires db deletions and re-insertion of new objects, do you really wanna do it ? 
i know i dont.